modules/reference/pages/properties/object-storage-properties.adoc

```diff
@@ -11,7 +11,7 @@ Object storage properties should only be set if you enable xref:manage:tiered-st
 
 === cloud_storage_access_key
 
-AWS or GCP access key. This access key is part of the credentials that Redpanda requires to authenticate with object storage services for Tiered Storage. This access key is used with the <<cloud_storage_secret_key>> to form the complete credentials required for authentication.
+Cluster provider access key. This access key is part of the credentials that Redpanda requires to authenticate with object storage services for Tiered Storage. This access key is used with the <<cloud_storage_secret_key>> to form the complete credentials required for authentication.
```

Comment:
Azure uses a separate property
`cloud_storage_azure_storage_account`
so this change is not really needed since `cloud_storage_access_key` is only for AWS or GCP